microprofile.ls/README.md

```diff
@@ -134,6 +134,28 @@ In other words, `mp.messaging.outgoing.generated-price.topic` exists only
 if there is the declaration `mp.messaging.outgoing.generated-price.connector=smallrye-kafka` 
 
 The `mp.messaging.outgoing.generated-price.topic` property cannot be computed on the external component side because it depends on the value of 
-`mp.messaging.outgoing.generated-price.connector`. The comput	tion is done on MicroProfile LS side with custom builder by using Java SPI [ItemMetadataProviderFactory](/src/main/java/org/eclipse/lsp4mp/extensions/ItemMetadataProviderFactory).
+`mp.messaging.outgoing.generated-price.connector`. The computation is done on MicroProfile LS side with custom builder by using Java SPI [ItemMetadataProviderFactory](https://github.com/eclipse/lsp4mp/blob/master/microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/extensions/ItemMetadataProviderFactory.java).
 
-Please see the [sample of MicroProfile Reactive Messaging](/src/main/java/org/eclipse/lsp4mp/extensions/reactivemessaging).
\ No newline at end of file
+Please see the [sample of MicroProfile Reactive Messaging](https://github.com/eclipse/lsp4mp/blob/master/microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/extensions/reactivemessaging).
+
+Contribute to settings
+-------
+
+In microprofile-config.properties file when a property doesn't exists, there is an unkwown error. The user can configure which unknown property errors to hide in the client settings.
+It's possible to ignore this error with contribution (ex : camel toolings can ignore validation for all `camel.*` properties).
+
+To do that in the JAR extension, create the `META-INF/lsp4mp/settings.json` like this:
```

Comment:
Mention creating a lsp4mp extension, link to the documentation for creating an lsp4mp extension. Then mention that the `META-INF/lsp4mp/settings.json` should be put into the lsp4mp extension.

Reply:
@datho7561 there is a new section `LSPMP - LS - extensions`, please tell me if it's better.

Reply:
It looks good!